src/pages/Home.jsx

```diff
@@ -11,89 +11,147 @@ import { GiScales } from 'react-icons/gi';
 import { ImCancelCircle } from 'react-icons/im';
 import Search from '../pages/Search'
 import Pets from './Pets';
+import { locations, animals } from '../data/pets'
 
 function Home() {
+    const [loading, setLoading] = useState(false)
+    const [pets, setPets] = useState([])
+    const [animal, setAnimal] = useState('')
+    const [location, setLocation] = useState('')
+    const FetchPets = async () => {
+        setLoading(true)
+        try {
+            
+            const response = await fetch(`https://pets-v2.dev-apis.com/pets?animal=${animal}&location=${location}`)
+            const data = await response.json()
+            setPets(data.pets)
+            setLoading(false)
+        } catch (error) {
+            throw error
+        }
+    }
+    useEffect(() => {
+        // const FetchPets = async () => {
+        //     setLoading(false)
+        //     try {
+        //         setLoading(true)
+        //         const response = await fetch(`https://pets-v2.dev-apis.com/pets?animal=${animal}&location=${location}`)
+        //         const data = await response.json()
+        //         setPets(data.pets)
+        //     } catch (error) {
+        //         throw error
+        //     }
+        //     setLoading(false)
+        // }
+        FetchPets()
+    }, 
+    // eslint-disable-next-line
+    [animal, location])
```

Comment:
I see that once you select pet or location it makes the API call immediately. This is nice if the search button is not there. You can either remove the search button, or make the functionality to work with the existent search button